pyccel/ast/cupyext.py

```diff
@@ -436,6 +444,39 @@ def __str__(self):
 
 #==============================================================================
 
+class CupyRavel(CupyArray):
+    """
+    Class representing a call to the cupy ravel function which
+    returns flattened version of the passed array
+
+    Parameters
+    ==========
+    arg : PyccelAstNode
+            A PyccelAstNode of unknown shape
+    memory_location : string
+            The location where the new array memory should be allocated
+    """
+    name   = 'ravel'
+    __slots__ = ()
+    _attribute_nodes = ('_arg',)
```

Comment:
Could you omit this line: `_attribute_nodes = ('_arg',)` and just let it be inherited from `CupyArray` to avoid risking it getting out of date?

Reply:
Done in [be2fd09](https://github.com/pyccel/pyccel/pull/1222/commits/be2fd091c234f25b4ceaddf4fb8ac361e6c16966).